src/operator/c_lapack_api.h

```diff
@@ -111,7 +111,11 @@ inline char loup(char uplo, bool invert) { return invert ? (uplo == 'U' ? 'L' :
  * \param lda leading dimension of a
  */
 template <typename xpu, typename DType>
-inline void flip(int m, int n, DType *b, int ldb, DType *a, int lda);
```

Comment:
What was the previous definition for flip?

Reply:
Only the `DType=float` and `DType=double` versions were implemented in the subsequent lines. I the function prototype in this line with a generalized version templated on the `DType`.

This change was necessary due to the use of `MSHADOW_TYPE_SWITCH`, I believe. Without this templating I was observing compile errors when testing with non `float` and `double` types.

I think the `float` and `double`-specific versions can be removed, actually...

Reply:
Yes they don't make much sense.

Please remove them

Reply:
Done. See 66afdc3 below.